.eslintrc.js

```diff
@@ -44,6 +44,12 @@ module.exports = {
             "error",
             "never"
         ],
-        "no-var": "error"
+        "no-var": "error",
+        // part of Node Style-guide but ignored
+        "max-len": 0,
+        "max-statements": 0,
+        "object-curly-spacing": 0,
```

Comment:
Let's change this to ["error", "always"]. We usually add padding to single line object literals.

Reply:
thanks for the help!

Reply:
Ok, added that, but it only is for object litterals.

for `if` and other flow control statements you need "curly" rule and in our case the "multi-or-nest" option: https://eslint.org/docs/rules/curly#multi-or-nest

That is what comes most close to the current code style (but it will enforce *not* using braces when there is only a single statement in an if or else, but enforces to nest it)